src/assets/scss/search.scss

```diff
@@ -25,14 +25,45 @@
     font-size: 14px;
     font-weight: normal;
   }
+  .header-action,
   .action {
-    width: 125px;
+    min-width: 100px !important;
+    position: sticky;
+    left: 0;
+    right: 0;
+    background-color: $BCgovInputBG;
   }
   .header-row-1 {
     th {
       font-size: 14px;
       font-weight: bold !important;
       // color: $BCgovBlack !important;
+      min-width: 130px;
+    }
+
+    // colum width
+    .header {
```

Comment:
fixed width for each column to get vertical scroll bar and show title in one line